src/axom/inlet/docs/sphinx/verification.rst

```diff
@@ -0,0 +1,54 @@
+.. _inlet_verification_page_label:
+
+############
+Verification
+############
+
+Before input file data can be accessed, it must first be verified by calling the ``verify()``
+method of the top-level ``Inlet`` object. This will return a ``bool`` indicating whether the
+provided input conformed to the schema and specific violations of the schema are logged via
+SLIC.
+
+This section describes the verification rules that apply to each possible element of the Inlet
+hierarchy, namely, ``Container``, ``Field``, and ``Function``.
+
+Container Verification
+----------------------
+
+If a ``Container`` is marked as required (via the ``required()``) method, then the ``Container`` must
+have a ``Field`` or ``Function`` that was present in the input or contain a sub-``Container`` that does.
```

Comment:
Can a container not be empty? In yaml, if you tell me `foo` is required, both `foo: {}` and `foo: []` should be viewed as valid.